static/app/views/monitors/components/monitorForm.tsx

```diff
@@ -1,7 +1,10 @@
-import {Fragment, useRef} from 'react';
+import {useRef} from 'react';
 import styled from '@emotion/styled';
 import {Observer} from 'mobx-react';
 
+import Alert from 'sentry/components/alert';
+import {RadioField} from 'sentry/components/forms';
```

Comment:
nit: can you import this directly instead of from the barrel class

Reply:
TIL its called a barrel class

Reply:
sorry module, but yeah barrel module